json-guides/retry-timeout.json

```diff
@@ -442,9 +444,9 @@
                                 "@ApplicationScoped",
                                 "public class BankService {",
                                 "",
-                                "    @Retry(retryOn=TimeoutException.class,",
-                                "           maxRetries=10,",
-                                "           maxDuration=4,",
+                                "    @Retry(retryOn = TimeoutException.class,",
+                                "           maxRetries=4,",
+                                "           maxDuration=10,",
                                 "           durationUnit=ChronoUnit.SECONDS,",
                                 "           delay=400, dealyUnit=ChronoUnit.MILLIS,",
```

Comment:
`delay` and `jitter` values were also changed to 200.

Typo on `dealyUnit` and `ChronoUnite` below

Reply:
Changes made.    Jitter was left to 100, per agreement with Kevin, because the default jitter is already 200.